xml/depl_nodes.xml

```diff
@@ -5403,23 +5445,23 @@ sudo openstack router add subnet <replaceable>router2</replaceable> priv-net-sub
      </para>
     </listitem>
    </varlistentry>
-   <varlistentry>
-    <term>cron_config</term>
-    <listitem>
-     <para>
-      Specifies when to run <systemitem>elasticsearch-curator</systemitem>.
-      Attributes of this parameter correspond to the fields in
-      <systemitem>crontab(5)</systemitem>.
-     </para>
-    </listitem>
-   </varlistentry>
   </variablelist>
 
   <bridgehead renderas="sect3"><guimenu>kafka: tunables for
 Kafka</guimenu>
   </bridgehead>
 
   <variablelist>
+   <varlistentry>
+    <term>config_options</term>
```

Comment:
Also here the parameter hierarchy is flattened. `replicas`, `partitions` and `config_options` are all options of `topics` list.
I think it's clear enough for the user though when she looks at the template https://github.com/crowbar/crowbar-openstack/blob/master/chef/data_bags/crowbar/template-monasca.json

Reply:
thank you for the link. makes things clearer.

changes made according to comments.